rllib/evaluation/sampler.py

```diff
@@ -304,6 +348,10 @@ def __init__(self,
         self.shutdown = False
         self.observation_fn = observation_fn
         self._use_trajectory_view_api = _use_trajectory_view_api
+        self.sample_collector = None
```

Comment:
If we use the TrajectoryView API, each RolloutWorker has-a SampleCollector object, which it uses for all policies/(fast) sample collection purposes.